File: frontend/src/app/model/board.ts
export class Board {
    public id : string;
    public name : string;
    public description : string;
    public ownerId : string;

    constructor(id : string, name : string, description : string, ownerId : string) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.ownerId = ownerId;
    }
}

File: backend/backend.Domain/Model/Board.cs
using backend.Domain.Model.Abstract;

namespace backend.Domain.Model
{
    public class Board : Entity
    {
        public string Name { get; set; }
        public string Description { get; set; }
        public Guid OwnerId { get; set; }

        public ICollection<Participant> Participants { get; set; }
    }
}
